uw-frame-components/portal/timeout/services.js

```diff
@@ -11,8 +11,9 @@ define(['angular', 'jquery'], function(angular, $) {
     * getTimeout() - Gets the shib session, but returns just the timeout
                      and the time of that timeout.
     **/
-    .factory('PortalShibbolethService', ['$http', 'miscService', 'SERVICE_LOC',
-        function($http, miscService, SERVICE_LOC) {
+    .factory('portalShibbolethService', ['$http', '$log',
```

Comment:
Interesting. I guess I'd assumed at this point that higher ed had turned the corner on Shibboleth adoption. Maybe [TIER](https://www.internet2.edu/vision-initiatives/initiatives/trust-identity-education-research/) will help with this. Anyway, started a [thread on uportal-user@](https://groups.google.com/a/apereo.org/d/topic/uportal-user/Y8Df_QUgJw0/discussion). 

When it comes right down to it, `uPortal-home` is targeting whoever's willing to adopt it. If there are would-be adopters who would adopt but for more flexible user attribute sourcing, then by all means, we'll add user attribute sourcing features. But I don't think uPortal having its own user attributes abstraction is the way of the future. SAML SPs and attribute authorities for the win.